tests/ui/ui_tester.py

```diff
@@ -141,31 +295,99 @@ def create_path(path):
         create_path(output_dir)
         output_dir = os.path.join(output_dir, self._test_name)
         create_path(output_dir)
-        output_dir = os.path.join(output_dir, self._get_browser())
+        output_dir = os.path.join(output_dir, self._get_browser_name())
         create_path(output_dir)
         return output_dir
 
-    def _get_browser(self):
-        return self._driver.capabilities['browserName']
+    def _get_browser_name(self):
+        return self.browser.capabilities['browserName']
 
 
 class WorksheetTest(UITester):
     def __init__(self):
-        super().__init__('worksheet')
+        super().__init__('worksheet_rendering')
```

Comment:
Can you rename it back to `worksheet` for this PR? That way, it will be easier to review the PR directly on github to make sure no screenshots have changed. You can rename it to `worksheet_rendering` in another PR and then that should just be a simple rename to review.

```suggestion
        super().__init__('worksheet')
```